src/calculateRentalCost.js

```diff
@@ -1,10 +1,22 @@
+'use strict';
 /**
  * @param {number} days
  *
  * @return {number}
  */
+
 function calculateRentalCost(days) {
-  // write code here
+  const rentalCost = days * 40;
+
+  if (days >= 7) {
+    return rentalCost - 50;
```

Comment:
Use a constant for the number of days that qualifies for a long-term discount, and another for the amount of the discount itself, to avoid hardcoding. For example, `const LONG_TERM_RENTAL = 7;` and `const LONG_TERM_DISCOUNT = 50;`.